docs/guide/breakpoint-preview.md

```diff
@@ -149,11 +159,14 @@ Advanced features of media queries, including experimental or non-standard ones,
 ### Summary of Unsupported or Partially Supported Features
 | **Media Query Feature**       | **Support in Container Queries**       |
 |-------------------------------|---------------------------------------|
-| `min-height` / `max-height`   | ❌ Unsupported                        |
-| `orientation`                 | ❌ Unsupported                        |
-| `aspect-ratio`                | ❌ Unsupported                        |
-| Complex logical conditions    | ⚠️ Partial / Unreliable               |
-| Viewport-relative units       | ⚠️ Not container-aware               |
-| User preference queries       | ❌ Unsupported (e.g., dark mode)      |
+| `orientation` | ❌ Unsupported |
```

Comment:
Sounds like orientation and aspect-ratio are still candidates to be supported at some point, per my ticket, but this is a good documentation update.